crossing-training.lic

```diff
@@ -228,8 +228,8 @@ class CrossingTraining
   end
 
   def check_nerves
-    wounds = check_health['wounds']
-    return unless wounds.any? { |_k, v| v.include?('skin') }
+    wounds = DRCH.check_health['wounds']
+    return unless wounds.values.flatten.any? { |wound| wound.body_part == 'skin' }
```

Comment:
Updates to new method name and data structure of return type.  `DRCH.check_health['wounds']` are now instances of `DRCH::Wound` class and one of its properties is `body_part`.